src/components/List/List.tsx

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import cn from 'classnames';
 
 import classes from './List.module.css';
-
+import type { BorderStyle } from './Context';
+import { ListContext } from './Context';
 export interface ListProps {
   children?: React.ReactNode;
+  borderStyle: BorderStyle;
```

Comment:
Are there cases where borderstyle on the root element and the list items are different, or could they both just read the value from context?

Reply:
There's no cases of them as far as I know, but I'll control check with design

Reply:
Design says there's no cases of different borderstyles, so reading the same value from context will be fine for now